skiko/src/jvmMain/cpp/common/stubs.cc

```diff
@@ -59,6 +59,11 @@ JNIEXPORT void JNICALL Java_org_jetbrains_skiko_context_Direct3DContextHandler_f
     skikoUnimplemented("Java_org_jetbrains_skiko_context_Direct3DContextHandler_flush");
 }
 
+JNIEXPORT jlong JNICALL Java_org_jetbrains_skiko_redrawer_Direct3DRedrawer_chooseAdapter(JNIEnv *env, jobject redrawer, jint adapterPriority) {
```

Comment:
Is it called anywhere? Why do we need this new function without an implementation?

Reply:
It should not be called, but we should have all native JNI references declared - so it's like no-op implementation for Linux where we don't have DirectX. I just fixed the inconsistency here, but probably it makes sense to revisit this approach some day.

Reply:
Got it